INFO:root:24-05-2021-14-16-06
INFO:root:loaded class labels:
INFO:root:water
INFO:root:land
INFO:root:24-05-2021-14-16-06
INFO:root:Folder created: results/results2021-05-24-14-16
INFO:root:24-05-2021-14-16-06
INFO:root:Results will be written to results/results2021-05-24-14-16
INFO:root:24-05-2021-14-16-06
INFO:root:loaded files:
INFO:root:assets/D800_20160308_222129lr02-0.jpg
INFO:root:assets/D800_20160308_222129lr02-2.jpg
INFO:root:assets/D800_20160308_222129lr02-3.jpg
INFO:root:assets/D800_20160308_222135lr03-0.jpg
INFO:root:assets/D800_20160308_222135lr03-1.jpg
INFO:root:assets/D800_20160308_222135lr03-2.jpg
INFO:root:assets/D800_20160308_222135lr03-3.jpg
INFO:root:assets/D800_20160308_222129lr03-2.JPG
INFO:root:assets/D800_20160308_222129lr03-3.JPG
INFO:root:assets/D800_20160308_222135lr00-2.JPG
INFO:root:assets/D800_20160308_222135lr02-0.jpeg
INFO:root:assets/D800_20160308_222135lr02-2.jpeg
INFO:root:assets/D800_20160308_222135lr02-3.jpeg
INFO:root:assets/D800_20160308_222136lr02-2.jpeg
INFO:root:assets/D800_20160308_222136lr02-3.jpeg
INFO:werkzeug: * Running on http://127.0.0.1:8050/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [24/May/2021 14:16:08] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [24/May/2021 14:16:08] "GET /_dash-layout HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [24/May/2021 14:16:08] "GET /_dash-dependencies HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [24/May/2021 14:16:08] "GET /_favicon.ico?v=1.12.0 HTTP/1.1" 200 -
INFO:root:24-05-2021-14-16-08
INFO:root:File list written to files_done.txt
ERROR:doodler_server:Exception on /_dash-update-component [POST]
Traceback (most recent call last):
  File "/home/marda/anaconda3/envs/dashdoodler/lib/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marda/anaconda3/envs/dashdoodler/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/marda/anaconda3/envs/dashdoodler/lib/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/marda/anaconda3/envs/dashdoodler/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/marda/anaconda3/envs/dashdoodler/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/marda/anaconda3/envs/dashdoodler/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/marda/anaconda3/envs/dashdoodler/lib/python3.6/site-packages/dash/dash.py", line 1032, in dispatch
    response.set_data(func(*args, outputs_list=outputs_list))
  File "/home/marda/anaconda3/envs/dashdoodler/lib/python3.6/site-packages/dash/dash.py", line 967, in add_context
    output_value = func(*args, **kwargs)  # %% callback invoked %%
  File "doodler_server.py", line 814, in update_output
    if 'assets' not in select_image_value:
UnboundLocalError: local variable 'select_image_value' referenced before assignment
INFO:werkzeug:127.0.0.1 - - [24/May/2021 14:16:08] "[35m[1mPOST /_dash-update-component HTTP/1.1[0m" 500 -
INFO:root:24-05-2021-14-16-08
INFO:root:File list written to files_done.txt
ERROR:doodler_server:Exception on /_dash-update-component [POST]
Traceback (most recent call last):
  File "/home/marda/anaconda3/envs/dashdoodler/lib/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marda/anaconda3/envs/dashdoodler/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/marda/anaconda3/envs/dashdoodler/lib/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/marda/anaconda3/envs/dashdoodler/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/marda/anaconda3/envs/dashdoodler/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/marda/anaconda3/envs/dashdoodler/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/marda/anaconda3/envs/dashdoodler/lib/python3.6/site-packages/dash/dash.py", line 1032, in dispatch
    response.set_data(func(*args, outputs_list=outputs_list))
  File "/home/marda/anaconda3/envs/dashdoodler/lib/python3.6/site-packages/dash/dash.py", line 967, in add_context
    output_value = func(*args, **kwargs)  # %% callback invoked %%
  File "doodler_server.py", line 814, in update_output
    if 'assets' not in select_image_value:
UnboundLocalError: local variable 'select_image_value' referenced before assignment
INFO:werkzeug:127.0.0.1 - - [24/May/2021 14:16:08] "[35m[1mPOST /_dash-update-component HTTP/1.1[0m" 500 -
INFO:root:24-05-2021-14-16-28
INFO:root:loaded class labels:
INFO:root:water
INFO:root:land
INFO:root:24-05-2021-14-16-28
INFO:root:Results will be written to results/results2021-05-24-14-16
INFO:root:24-05-2021-14-16-28
INFO:root:loaded files:
INFO:root:assets/D800_20160308_222129lr02-0.jpg
INFO:root:assets/D800_20160308_222129lr02-2.jpg
INFO:root:assets/D800_20160308_222129lr02-3.jpg
INFO:root:assets/D800_20160308_222135lr03-0.jpg
INFO:root:assets/D800_20160308_222135lr03-1.jpg
INFO:root:assets/D800_20160308_222135lr03-2.jpg
INFO:root:assets/D800_20160308_222135lr03-3.jpg
INFO:root:assets/D800_20160308_222129lr03-2.JPG
INFO:root:assets/D800_20160308_222129lr03-3.JPG
INFO:root:assets/D800_20160308_222135lr00-2.JPG
INFO:root:assets/D800_20160308_222135lr02-0.jpeg
INFO:root:assets/D800_20160308_222135lr02-2.jpeg
INFO:root:assets/D800_20160308_222135lr02-3.jpeg
INFO:root:assets/D800_20160308_222136lr02-2.jpeg
INFO:root:assets/D800_20160308_222136lr02-3.jpeg
INFO:werkzeug: * Running on http://127.0.0.1:8050/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [24/May/2021 14:16:30] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [24/May/2021 14:16:30] "GET /_dash-layout HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [24/May/2021 14:16:30] "GET /_favicon.ico?v=1.12.0 HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [24/May/2021 14:16:30] "GET /_dash-dependencies HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [24/May/2021 14:16:43] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [24/May/2021 14:16:43] "GET /_dash-layout HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [24/May/2021 14:16:43] "GET /_dash-dependencies HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [24/May/2021 14:16:43] "GET /_favicon.ico?v=1.12.0 HTTP/1.1" 200 -
